difference.go

```diff
@@ -1,26 +1,35 @@
 package difference
 
 import (
+	"errors"
 	"math"
 )
 
 /*
-This function takes absolute change between two numbers, dividing it by the initial value,
+This function takes change between two numbers, dividing it by absolute value of initial value,
 and multiplying it by 100.
+If result is positive, it means there is an increase.
+If result is negative, it means there is a decrease.
+Initial value cannot be zero.
 */
-func PercentChange(initial float64, final float64) float64 {
-	diff := math.Abs(final - initial)
-
-	return diff / initial * 100
+func PercentChange(initial float64, final float64) (float64, error) {
+	if initial == 0 {
```

Comment:
I think we can check for above zero here instead like `>0`

Reply:
eventually we will want to be able to handle zero here

Reply:
PercentChange should be able to handle all values expect zero value. There is no percent change for zero value. 
If zero is denominator, the answer is infinity. It does not make sense mathematically.

Reply:
I see, so negative numbers are ok?

Reply:
I think negative numbers should be fine. We can calcualate percent changes with negative value as we take absolute of initial value as reference.  I use this link https://www.calculatorsoup.com/calculators/algebra/percent-change-calculator.php to check calcualation.